paraview/src/CloudberryDemo.js

```diff
@@ -0,0 +1,437 @@
+/* global document */
```

Comment:
if that is the case, can you add the script that can generate this files instead?